docs/onboarding_to_cluster.md

```diff
@@ -0,0 +1,151 @@
+# Onboarding to a cluster
+
+Currently we support 2 ways to request/perform onboarding to any supported clusters:
+
+1. Onboarding request - please use the **Onboarding to Cluster** issue template in [operate-first/support](https://github.com/operate-first/support) repository.
+2. Opening a PR with the desired changes. The rest of this doc focuses on this second option. We'd gladly welcome your contributions.
+
+## Prerequisites
```

Comment:
a conditional pre-requisite  is a gpg key & fingerprint of that key if they are adding a new group for that cluster.

Reply:
although I'm noticing that these are included in the issue-template, I'll leave it to your discretion whether we should mention that here. Also worth considering are things like 

- Cluster -> namespace(s) to add
- user(s) to add

Reply:
Yeah, I've listed only the technical requirements. I'll add that. LMKWYT